File: Csharp/2011.cs
public class Solution {
    public int FinalValueAfterOperations(string[] operations) {
        int x = 0;
        for(int i = 0; i < operations.Length; i++){
            if(operations[i] == "++X"){
                ++x;
            } else if(operations[i] == "X++"){
                x++;
            } else if(operations[i] == "--X"){
                --x;
            } else if(operations[i] == "X--"){
                x--;
            }
        }
        return x;
    }
}

File: JS/2011.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function(operations) {
    let x = 0
    for(let i = 0; i < operations.length; i++){
        if(operations[i] === "++X" || operations[i] === "X++") x++
        else x--
    }

    return x
};